app/models/favorite.rb

```diff
@@ -17,12 +17,14 @@ def favorite_count
     @content.values.sum
   end
 
-  def favorite_delete(id)
-    @content.delete(id.to_s)
+  def favorite_delete(pets)
+    pets.each do |pet|
+      @content.delete(pet.id.to_s)
+    end
   end
 
   def keys
-    @content.map {|key, value| key.to_i }
+    @content.keys 
```

Comment:
Refactors the keys method- not necessary to convert them to an integer like I thought we had to